cis/Issues/digitaalPaspoortVerzekeren/description.xml

```diff
@@ -0,0 +1,97 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo Stichting Centraal Informatie Systeem</en>
+		<nl>Demo Stichting Centraal Informatie Systeem</nl>
+	</Name>
+	<ShortName>
+		<en>Demo CIS</en>
+		<nl>Demo CIS</nl>
+	</ShortName>
+	<SchemeManager>irma-demo</SchemeManager>
+	<IssuerID>cis</IssuerID>
+	<CredentialID>digitaalPaspoortVerzekeren</CredentialID>
```

Comment:
In the previous PR you had an English name as `CredentialID`, and now a Dutch one. Was this a deliberate choice?

Reply:
it was but now that I look at the other issuers we will stick with the english one for now.